source/settings/core.rst

```diff
@@ -1433,9 +1433,9 @@ See :ref:`settings` for list of all setting groups.
 
 .. dovecot_core:setting:: listen
    :default: \*, \:\:
-   :values: @ip_addresses
+   :values: @boollist
 
-   A comma-separated list of IP addresses or hostnames on which external network
+   A space-separated list of IP addresses or hostnames on which external network
```

Comment:
I think shouldn't say anymore that it's a "space-separated list". The boollist can be, but doesn't have to be. So I think here could be just "A list of IP addresses..."